gitclone/git.go

```diff
@@ -92,23 +91,6 @@ func isPrivate(repoURL string) bool {
 	return strings.HasPrefix(repoURL, "git")
 }
 
-// If incoming branch matches to pull/x/merge pattern headBranchRefs
```

Comment:
Its only usage was the previous merge branch checkout strategy, and I don't think it's correct.